client/client_pool/src/external_client.cpp

```diff
@@ -227,16 +218,10 @@ void ConcordClient::CreateClient(ConcordClientPoolConfig& config, const SimpleCl
   BaseCommConfig* comm_config = CreateCommConfig(num_replicas, config);
```

Comment:
We need to avoid this part if we are using multiplex comm.

Also the part in lines 238-239 where we create the communication layer.